home/templates/home/index.html

```diff
@@ -441,7 +441,7 @@ <h4 class="feature_sub">Events we conduct </h4>
             <div class="col-md-12 col-sm-12 col-xs-12">
                 <div class="feature_header text-center">
                     <h3 class="feature_title">What <b>We do</b></h3>
-                    <h4 class="feature_sub">Lorem ipsum dolor sit amet, consectetur adipisicing elit. </h4>
+                    <h4 class="feature_sub" align="center"> We are a group of Open Source Enthusiasts who focus on something more preliminary and relevant, ”A Love for Coding”.<br>KOSS plans to hold events such as familiarizing students with terminal commands, Linux environment, Github, teach algorithms and data structures, besides conducting workshops on GUI programming, Web development , Android applications and Hackathons.</h4>
```

Comment:
> What does VCS have to do with gh handles?

There is no point in talking about GitHub when the person is not familiar with Git.
Also, GitHub ain't the only Git-based platform for hosting software.
So, VCS are a more relevant topic here.

Moreover, I think `Git Development Workflow` would be a more suitable title.